ParseLiveQuery/src/test/java/com/parse/TestParseLiveQueryClient.java

```diff
@@ -390,16 +390,6 @@ public void testEmptySessionTokenOnSubscribe() {
     }
 
     @Test
-    public void testDisconnectOnBackgroundThread() throws Exception {
-        executor.pause();
```

Comment:
It may be useful in the future if we need to test anything that happens asynchronously (like websocket connect/disconnect events, or something like that).

It will likely need to go away if/when #36 is done, because it won't be using an Executor anymore.

I can remove it for now if you think it's better not to have it if we're not using it?